src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int LENGTH_NUMBER = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall(Ball ball) {
+        ball.setColor(colorSupplier.getRandomColor());
+        int randomNumber = random.nextInt(LENGTH_NUMBER);
+        ball.setNumber(randomNumber);
```

Comment:
no need for extra variable
```suggestion
        ball.setNumber(random.nextInt(MAX_NUMBER));
```